### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by MatHenriquez**
# Summary
- Delete old home page content.
- Create a container with a grid layout for new content.
- Add a navbar with the wallet brand.
- Add sign in and sign up buttons.
- Add basic styles to the new home page content.

# Details
- Create the template for the new login function on the home page: sign in with an existing key or create a new key.

# Evidence
## Updated Home Page
updated_home_page

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -1,7 +1,34 @@
-export default function Index(){
-    return(
-        <h1>
-            Stellar Wallet
-        </h1>
-    )
-}
\ No newline at end of file
+export default function Index() {
```
Comment (Genebson) [inline app/page.tsx]: This has to be typed as a component.

File: app/page.tsx [hunk 2]
```diff
@@ -1,7 +1,34 @@
...
+  return (
+    <div className="grid grid-cols-1 grid-rows-6 h-screen bg-cyan-950">
+        
+      {/* Brand */}
```
Comment (Genebson) [inline app/page.tsx]: Remember that it's not necessary to add comments to the code.

**Review by Genebson — APPROVED** [review]

File: app/page.tsx (hunk 1 above)
Comment (MatHenriquez) [reply app/page.tsx]: I have already committed this change and now the Index has the React functional component type

File: app/page.tsx (hunk 2 above)
Comment (MatHenriquez) [reply app/page.tsx]: Thank you, I have removed these comments and commited the changes.

**Review by Genebson — APPROVED** [review]

**Pull request merged**